cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -38,11 +38,11 @@
     {% set video_id = options.video.id | default('') %}
 
     {# TODO: Replace with django method for getting video id if youtube and urlparams #}
```

Comment:
yea, I'll update the TODO to use simpler language. So I'm refactoring the logic out of the template in https://github.com/cfpb/cfgov-refresh/commit/17a7c32e61282db65bbad76d5791ed4bbee5e252 but will submit that in separate PR.